modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -92,10 +57,9 @@ const renderStatusbar = (editor: Editor, providersBackstage: UiFactoryBackstageP
   const getComponents = (): SimpleSpec[] => {
```

Comment:
This could be simplified to this maybe?
```
  const getComponents = (): SimpleSpec[] => {
    const resizeHandler = ResizeHandler.renderResizeHandler(editor, providersBackstage);
    return getTextComponents().concat(resizeHandler.toArray());
  }
```